docker/entrypoint.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Define the start command
+START_COMMAND="teosd"
+
+# Set the API bind address
+if [[ ! -z ${API_BIND} ]]; then
+    START_COMMAND="$START_COMMAND --apibind $API_BIND"
+fi
+
+# Set the API port
+if [[ ! -z ${API_PORT} ]]; then
+    START_COMMAND="$START_COMMAND --apiport $API_PORT"
+fi
+
+# Set the RPC bind address
+if [[ ! -z ${RPC_BIND} ]]; then
+    START_COMMAND="$START_COMMAND --rpcbind $RPC_BIND"
+fi
+
+# Set the RPC port
+if [[ ! -z ${RPC_PORT} ]]; then
+    START_COMMAND="$START_COMMAND --rpcport $RPC_PORT"
+fi
+
+# Set the Bitcoin network
+if [[ ! -z ${BTC_NETWORK} ]]; then
+    START_COMMAND="$START_COMMAND --btcnetwork $BTC_NETWORK"
+fi
+
+# Set the Bitcoin RPC credentials
+if [[ ! -z ${BTC_RPC_USER} ]]; then
+    START_COMMAND="$START_COMMAND --btcrpcuser $BTC_RPC_USER"
+fi
+
+if [[ ! -z ${BTC_RPC_PASSWORD} ]]; then
+    START_COMMAND="$START_COMMAND --btcrpcpassword $BTC_RPC_PASSWORD"
+fi
+
+# Set the Bitcoin RPC connection details
+if [[ ! -z ${BTC_RPC_CONNECT} ]]; then
+    START_COMMAND="$START_COMMAND --btcrpcconnect $BTC_RPC_CONNECT"
+fi
+
+if [[ ! -z ${BTC_RPC_PORT} ]]; then
+    START_COMMAND="$START_COMMAND --btcrpcport $BTC_RPC_PORT"
+fi
+
+# Set the TEOS data directory
+if [[ ! -z ${TEOS_DATADIR} ]]; then
+    START_COMMAND="$START_COMMAND --datadir $TEOS_DATADIR"
+fi
```

Comment:
I think we should omit this argument so not to confuse the user where that data directory should be (inside or outside docker, I think users will try to supply a data directory that's on the host and not the container). One default data directory that is volume mounted should be enough already.

Reply:
Indeed. That's actually how we used to do it for `python-teos`:

https://github.com/talaia-labs/python-teos#running-teos-in-a-docker-container

Reply:
wow!! Great idea. Thanks